app/models/ems_event.rb

```diff
@@ -84,6 +84,10 @@ def self.add_kubernetes(ems_id, event)
     add(ems_id, ManageIQ::Providers::Kubernetes::ContainerManager::EventParser.event_to_hash(event, ems_id))
   end
 
+  def self.add_middleware(ems_id, event)
```

Comment:
Two things here:

1. change to `add_hawkular` to capture the same scope that the other methods use (see `add_azure` and `add_kubernetes`).
2. @durandom did you have a common way of extracting these?  Should that be done here, or wait until we get to extracting all providers?